wordpress-mobile/WordPress-iOS
# Stats: replace No Results view.

Fixes #9988 

In Stats, replace `WPNoResultsView` with `NoResultsViewController`.

The No Results view is only displayed when there is no connection.

To test:
- Disconnect internet.
- Go to Stats.
- Verify the view is:
![no_connection](https://user-images.githubusercontent.com/1816888/44173563-8cf55c80-a09d-11e8-8865-92a3775dc4d2.png)
